libs/core/iterator_support/tests/unit/is_sentinel_for.cpp

```diff
@@ -21,15 +23,15 @@ void is_sentinel_for()
     HPX_TEST_MSG(
         (hpx::traits::is_sentinel_for<std::int64_t, std::int64_t>::value ==
             false),
-        "Integer - integer pair is ncompatible pair");
+        "Integer - integer pair is incompatible pair");
```

Comment:
Not really part of this PR, but isn't
```
HPX_TEST_MSG(
        !hpx::traits::is_sentinel_for<std::int64_t, std::int64_t>::value,
        "Integer - integer pair is incompatible pair");
```
more idiomatic?

Reply:
Done